gamemodes/cinema/gamemode/modules/theater/services/sh_youtube.lua

```diff
@@ -33,61 +33,74 @@ if CLIENT then
 	]]
-    local loaded = false
 
     function SERVICE:LoadVideo(Video, panel)
-        panel:EnsureURL("http://swamp.sv/s/cinema/" .. self:GetClass() .. ".html")
-        panel.vid = Video
-        local enabled = false
 
         if (self:IsMature(Video)) then
-            panel:EnsureURL("https://www.youtube.com/embed/" .. Video:Key() .. "?autoplay=1&controls=0&showinfo=0&modestbranding=1&rel=0&iv_load_policy=3&enablejsapi=1")
-            enabled = true
-            loaded = false
+			if (string.StartWith(panel:GetURL(),"http://swamp.sv/s/cinema/youtube.html")) then
+				theater.LoadVideo(Video,true)
```

Comment:
what is this supposed to do, recreate the panel?

Reply:
yes